pass_import/formats/kdbx.py

```diff
@@ -161,7 +161,7 @@ def insert(self, entry):
         group = os.path.dirname(path)
 
         root_group = self.keepass.root_group
-        kpgroup = self.keepass.find_groups(path=group)
+        kpgroup = self.keepass.find_groups(name=group, first=True)
```

Comment:
Looks like the doc of `find_groups()` doesn't match what the code actually expects. I totally agree with @roddhjav that using `name` is not possible: it would only work with one level of groups. The `path` parameter expects an array.

This would work better:

```suggestion
        kpgroup = self.keepass.find_groups(path=group.split("/"), first=True)
```

However before going this direction, it probably is worth waiting for the decision on https://github.com/libkeepass/pykeepass/issues/313